index.bs

```diff
@@ -371,6 +374,28 @@ To <dfn abstract-op>initialize a global object's embedder policy from a response
 
 </div>
 
+### Process a navigation fetch ### {#process-a-navigation-fetch}
+<div algorithm="process a navigation fetch">
+For every navigation request for a nested browsing context whose parent's embedder
+policy is "`require-corp`", CORP is checked.
+
+Run the following step before the sixth step of [=process a navigate fetch=].
+
+6.  If |browsingContext| is a [=child browsing context=]:
+
+    1.  Let |parent| be |browsingContext|'s [=parent browsing context=].
+
+    2.  Let |requestForCORPCheck| be a copy of |request|.
+
+    3.  Set |requestForCORPCheck|'s origin to |parent|'s origin.
+
+    4.  Set |requestForCORPCheck|'s client to |parent|'s [=active document=].
+
+    5.  If the result of [$cross-origin resource policy check$] with |requestForCORPCheck| and
+        |response| is `blocked`, then set |response| to a [=network error=].
+
```

Comment:
It's worth adding a note here about why we're running with the parent browsing context rather than the source browsing context so that it's clearly an intentional decision.

Reply:
Done.